frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/GraphElementManager.java

```diff
@@ -3,15 +3,13 @@
 import java.util.Comparator;
 
 import com.intellij.vcs.log.graph.api.elements.GraphElement;
-import com.intellij.vcs.log.graph.api.printer.PrintElementManager;
 import com.intellij.vcs.log.graph.impl.print.GraphElementComparatorByLayoutIndex;
-import com.intellij.vcs.log.graph.impl.print.elements.PrintElementWithGraphElement;
 import lombok.Getter;
 
 import com.virtuslab.gitmachete.frontend.graph.coloring.ColorGetterByLayoutIndex;
 import com.virtuslab.gitmachete.frontend.graph.repository.RepositoryGraph;
 
-public class GraphElementManager implements PrintElementManager {
+public class GraphElementManager {
   @Getter
   private final Comparator<GraphElement> graphElementComparator;
```

Comment:
Is this field needed now/was is even needed before?

Reply:
removed